draft-ietf-dnsop-svcb-https.md

```diff
@@ -1071,11 +1071,15 @@ Because HTTPS RRs are received over an often insecure channel (DNS),
 clients MUST NOT place any more trust in this signal than if they
 had received a 307 redirect over cleartext HTTP.
 
+When an HTTPS connection fails due to an error in the underlying secure
+transport, such as an error in certificate validation, some clients
+currently offer a "user recourse" that allows the user to bypass the
+security error and connect anyway.
 When making an "https" scheme request to an origin with an HTTPS RR,
-either directly or via the above redirect, the client SHOULD terminate the
-connection if there are any errors with the underlying secure transport, such as
-errors in certificate validation. This aligns with Section 8.4 and Section 12.1
-of {{HSTS}}.
+either directly or via the above redirect, a client that normally offers
+such a user recourse MAY remove it.  Origins that publish HTTPS RRs
```

Comment:
Is there some wording that doesn't involve the "it" which is a little ambiguous?
Perhaps "a client MAY choose to not provide user recourse".

Reply:
Rephrased